packages/api-v4/src/profile/permissions.ts

```diff
@@ -0,0 +1,15 @@
+import { API_ROOT } from '../constants';
+import Request, { setMethod, setURL } from '../request';
+import { ProfilePermissions } from './types';
+/**
+ * getPermissions
+ *
+ * Return all permissions for user's profile.
+ *
+ */
+export const getPermissions = () => {
```

Comment:
This API is going to be deleted. We'll just use the /account/users/username/permissions API https://docs.google.com/document/d/1Cg-EXXMcL3zy4wxPhmXmUgyRrK0BvTe0GFwLp2qMZ54/edit#heading=h.k9ywyzyp9kd8